### DISCUSSION TITLE
A problem about download compounds from the template library

### DISCUSSION THREAD

**Discussion by ZML0113** [Open-Systems-Pharmacology/Forum#1313] [Models]
I couldn't download a drug PBPK model from the template library, and the following screen is displayed:
![11111](https://user-images.githubusercontent.com/81542635/191159225-9b98e86b-c957-4781-ae75-fc3f2ccb76f4.jpg)

**Comment by msevestre** [comment]
Can you type in the url in your Webbrowser? What do you see?

**Reply by ZML0113** [reply]
![222](https://user-images.githubusercontent.com/81542635/191279445-f4f4a470-410b-4ad2-95f6-9593bef9d9af.jpg)
The attached graph was what I can see in the Webbrowser.

**Reply by msevestre** [reply]
so it looks like you can access this file 
And the error from above always shows up? For all molecules?

**Reply by ZML0113** [reply]
Yes, I couldn't download all the molecules

**Comment by GanleePK** [comment]
I had the same problem when download the template, perhaps the network firewalls stops it. You can download the compound models from OSP directly, and the URL was shown below:
https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library
